lib/Test/Nginx/Util.pm

```diff
@@ -915,6 +916,18 @@ sub setup_server_root ($) {
         bail_out "Failed to do mkdir $ConfDir\n";
 }
 
+sub setup_server_env ($) {
```

Comment:
The setup_server_env is not a good name either.

Reply:
Agreed, but I haven't come up with a good function name yet, and using functions is just to minimize redundant code.

Reply:
Any suggestion for this? or directly copy the function code and replace it at its original location?

Reply:
how about run_customer_init_script